exporter/signalfxexporter/translation/converter.go

```diff
@@ -208,28 +222,79 @@ func metricDataToSfxDataPoints(
 	return sfxDataPoints, numDroppedTimeSeries
 }
 
-func appendAttributesToDimensions(
-	dimensions []*sfxpb.Dimension,
-	attribs map[string]string,
-) []*sfxpb.Dimension {
+// sanitizeDataPointLabels replaces all characters unsupported by SignalFx backend
```

Comment:
Is it just reshuffling of the functions? can it be another commit if we really need it? it makes it hard to review

Reply:
I moved the functions over to the new file one by one, but yeah, the diff that GitHub created ended up kind of garbled. I'll see if I can make this simpler.

Reply:
Fixed.

Reply:
Oh I didn't notice it's a new file